### DISCUSSION TITLE
Gpx without note option

### DISCUSSION THREAD

**Discussion by Martinligabue** [Helium314/SCEE#514] [Ideas]
I think it would be nice to have the option to disable the requirement for creating a note after generating a gpx

**Comment by mnalis** [comment]
If you don't want to create public OSM Notes, you can enable `Setttings` / `SCEE settings` / `Notes` option `Show GPX note button`. Then, when being prompted to create note, you'll have additional button to create only GPX instead of public OSM Note.

Does that help?

**Reply by Helium314** [reply]
This way you would still need to create a note, but at least it's not public.

**Reply by Martinligabue** [reply]
That is good, I would recommend making it a bit clearer how to get there, as I asked in my local group and searched faqs and online and no-one knew whow to do it

**Reply by mnalis** [reply]
Glad that helped, @Martinligabue! 

But yes, you are correct, SCEE (even if it has _"Expert Edition"_ in its acronym) would benefit from some more extensive documentation than just the [README.md](https://github.com/Helium314/SCEE/blob/modified/README.md#differences-to-streetcomplete) which has somewhat sparse _"Create personal notes in a GPX file (adds a new button when creating a note)"_. And it does not have so big userbase, so not many people know all of its features. Best way I found is to jump into `SCEE settings` and look around what sounds interesting. :smiley: 

There is also public wiki for [SCEE](https://wiki.openstreetmap.org/wiki/SCEE), so if you feel like contributing some FAQ entries there, that would be welcome! But even just this discussion here might help other users searching for that specific issue - so thanks for asking!
